js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1527,7 +1527,7 @@ const EditingController = modules.ViewController.inherit((function() {
                 }
                 when(this._beforeSaveEditData()).done(cancel => {
                     if(cancel) {
-                        afterSaveEditData();
+                        afterSaveEditData(true);
```

Comment:
```suggestion
                        afterSaveEditData(cancel);
```
Since **afterSaveEditData** accepts the **cancel** parameter and you call it with **true** when saving is canceled, pass **cancel** to the function instead of **true**.